README.asciidoc

```diff
@@ -0,0 +1,61 @@
+== Mainboard communication
```

Comment:
README needs to be in `firmware` directory.

https://ut-robotics.github.io/picr22-home/team_documentation_requirements.html
https://ut-robotics.github.io/picr22-home/github_guide.html

Reply:
@markus-marandi this is not yet resolved.